environment-dev.yml

```diff
@@ -31,7 +31,10 @@ dependencies:
   - cyrus-sasl
   - aws-sdk-cpp >=1.11.405
   - prometheus-cpp
-  - libprotobuf <7
+    # We have to use the build of libprotobuf starting from this PR:
+    # https://github.com/conda-forge/libprotobuf-feedstock/pull/277
+  - sel(win): libprotobuf==5.29.3=h*_3
```

Comment:
This is a known installable and patched version of `libprotobuf 5`.

`libprotobuf 6` is not as installable for Windows in my experience, and also would need to be patched (see https://github.com/conda-forge/libprotobuf-feedstock/issues/275#issuecomment-3554088930).